cni/network/network.go

```diff
@@ -642,7 +642,7 @@ func (plugin *NetPlugin) createNetworkInternal(
 		ServiceCidrs:                  ipamAddConfig.nwCfg.ServiceCidrs,
 	}
 
-	if err = addSubnetToNetworkInfo(ipamAddResult, nwInfo); err != nil {
+	if nwInfo, err = addSubnetToNetworkInfo(ipamAddResult, nwInfo); err != nil {
```

Comment:
why do we need to return nwInfo as a result? Can't we pass the nwInfo as reference and modify that?